ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/ovn/sdn2ovn/provision/cucushift-installer-rehearse-vsphere-ipi-ovn-sdn2ovn-provision-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-ipi-ovn-sdn2ovn-provision
+  steps:
+  - chain: ipi-conf-vsphere
+  - chain: ipi-install-vsphere
+  - ref: ovn-sdn-migration
```

Comment:
Per yesterday's meeting, suggest to append ref: mirror-images-enable-qe-catalogsource here.

Reply:
Thanks, updated.

Reply:
after append `ref: mirror-images-enable-qe-catalogsource`, I got the following error, @jianlinliu @jianzhangbjz do you know this error?

```
ERRO[2022-06-29T08:29:57Z] Some steps failed:                           
ERRO[2022-06-29T08:29:57Z] 
  * could not run steps: step installer-rehearse-vsphere failed: "installer-rehearse-vsphere" pre steps failed: "installer-rehearse-vsphere" pod "installer-rehearse-vsphere-mirror-images-enable-qe-catalogsource" failed: the pod ci-op-bjhzmm4s/installer-rehearse-vsphere-mirror-images-enable-qe-catalogsource failed after 16s (failed containers: test): ContainerFailed one or more containers exited

```
in

Reply:
```
no proxy setting.
Running Command: oc whoami
system:admin
Running Command: oc version
Client Version: v4.2.0-alpha.0-1542-g5908f36
Kustomize Version: v4.5.4
Server Version: 4.11.0-0.nightly-2022-06-28-160049
Kubernetes Version: v1.24.0+9ddc8b1
head: cannot open '/tmp/secret/mirror_registry_url' for reading: No such file or directory
```

Reply:
Yeah, this is mentioned in https://github.com/openshift/release/pull/29896/files#r909453906, seem like `mirror-images-enable-qe-catalogsource` is not ready to be enabled in all install workflow.

Reply:
@zhaozhanqi Let us drop mirror-images-enable-qe-catalogsource again before it is ready to be integrated to avoid blocking your development, sorry for the inconvenience.

Reply:
checking

Reply:
@zhaozhanqi Now, you can enable QE CatalogSource by adding the below step:

- enable-qe-catalogsource (for connected env)

- enable-qe-catalogsource-disconnected (for disconnected env)

Reply:
@jianzhangbjz thanks, will have a try